src/scvmm/azext_scvmm/commands.py

```diff
@@ -0,0 +1,137 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=too-many-statements
+
+from azure.cli.core import AzCommandsLoader
+
+from ._client_factory import (
+    cf_vmmserver,
+    cf_cloud,
+    cf_virtual_network,
+    cf_virtual_machine_template,
+    cf_virtual_machine,
+    cf_availability_sets,
+    cf_inventory_items,
+)
+
+from ._validators import (
+    validate_param_combos,
+    validate_param_combos_for_vm,
+    validate_param_combos_for_avset,
+)
+
+
+def load_command_table(self: AzCommandsLoader, _):
+
+    with self.command_group('scvmm vmmserver', client_factory=cf_vmmserver) as g:
+        g.custom_command('connect', 'connect_vmmserver', supports_no_wait=True)
+        g.custom_command('update', 'update_vmmserver', supports_no_wait=True)
+        g.custom_command('delete', 'delete_vmmserver', supports_no_wait=True)
+        g.custom_show_command('show', 'show_vmmserver')
+        g.custom_command('list', 'list_vmmserver')
+
+    with self.command_group('scvmm cloud', client_factory=cf_cloud) as g:
+        g.custom_command(
+            'create',
+            'create_cloud',
+            supports_no_wait=True,
+            validator=validate_param_combos,
+        )
+        g.custom_command('update', 'update_cloud', supports_no_wait=True)
+        g.custom_command(
+            'delete', 'delete_cloud', supports_no_wait=True, confirmation=True
+        )
+        g.custom_show_command('show', 'show_cloud')
+        g.custom_command('list', 'list_cloud')
+
+    with self.command_group(
+        'scvmm virtual-network', client_factory=cf_virtual_network
+    ) as g:
+        g.custom_command(
+            'create',
+            'create_virtual_network',
+            supports_no_wait=True,
+            validator=validate_param_combos,
+        )
+        g.custom_command('update', 'update_virtual_network', supports_no_wait=True)
+        g.custom_command(
+            'delete', 'delete_virtual_network', supports_no_wait=True, confirmation=True
+        )
```

Comment:
Please add "wait" commands for this command group

Reply:
Added wait command